app/src/main/java/com/sopt/now/presentation/auth/login/LoginViewModel.kt

```diff
@@ -0,0 +1,43 @@
+package com.sopt.now.presentation.auth.login
+
+import android.os.Build.VERSION_CODES.N
+import android.util.Log
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.sopt.now.model.login.RequestLoginDto
+import com.sopt.now.model.login.ResponseLoginDto
+import com.sopt.now.utils.ApiFactory
+import com.sopt.now.utils.ErrorResponse
+import com.sopt.now.utils.NetworkUtil
+import com.sopt.now.utils.ServicePool.loginService
+import kotlinx.coroutines.launch
+
+class LoginViewModel : ViewModel() {
+    private var _status = MutableLiveData<Boolean>()
+    var status: MutableLiveData<Boolean> = _status
```

Comment:
```suggestion
    private val _status = MutableLiveData<Boolean>()
    val status: LiveData<Boolean> get()= _status
```
1. 우선 var + Mutable의 경우 가변 + 가변이라 사용하지 않는 것이 좋습니다!

2. 또 backing property에 있어 기존에  사용하던 방식( = _status )은 객체에 대한 참조를 가지고 있습니다. 따라서 status.value = “ “와 같이 내부 상태를 변경할 수 있게 됩니다.   
  
  제안드리는 두번째 방식의 경우
 _status 에 대한 참조를 매번 새롭게 얻는 방식입니다. 따라서 status를 통해 _status의 값을 변경할 수 없습니다. get()매서드가 LiveData 타입을 반환하기 때문이죠!